attributes/default.rb

```diff
@@ -6,12 +6,13 @@
 default['amazon-ecs-agent']['tag'] = 'latest'
 default['amazon-ecs-agent']['user'] = 'ubuntu'
 default['amazon-ecs-agent']['storage_driver'] = 'overlay'
-default['amazon-ecs-agent']['docker']['version'] = '1.11.2'
+default['amazon-ecs-agent']['docker']['version'] = '17.05.0'
```

Comment:
It's about a one year jump. According to Docker the last breaking change was 1.10, so I guess we'll see how it plays out in practice